src/devices/MAX31856/MAX31856.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Spi;
+using System.IO;
+using UnitsNet;
+
+namespace Iot.Device.Max31856
+{
+    /// <summary>
+    /// Documentation https://datasheets.maximintegrated.com/en/ds/MAX31856.pdf
```

Comment:
```suggestion
    /// MAX31856  - cold-junction compensated thermocouple to digital converter
```

Datasheet link should be in the README.md